srcCpp/PerftestTransport.cxx

```diff
@@ -665,6 +677,12 @@ std::string PerftestTransport::helpMessageString()
 << "\t-nic <ipaddr>                 - Use only the nic specified by <ipaddr>.\n"
 << "\t                                If not specified, use all available\n"
 << "\t                                interfaces\n"
+<< "\t-multicast <address>          - Use multicast to send data.\n"
+<< "\t                                Default not to use multicast\n"
+<< "\t                                <address> is optional, if unspecified:\n"
+<< "\t                                                latency 239.255.1.2,\n"
```

Comment:
@javmorcas  Why do we have the logic of a multicast address per Topic?
I feel that it adds complexity and at the end, if the user wants to set multicast address it is possible only one for all the topic. Maybe we need to analyze if it is necessary or not

Reply:
For debugging purposed they decided to use 3 different ones. (It looks way cleared on Wireshark, for example).